packages/flame/lib/src/layout/align_component.dart

```diff
@@ -48,41 +48,61 @@ import 'package:vector_math/vector_math_64.dart';
 /// );
 /// ```
 class AlignComponent extends PositionComponent {
-  /// Creates a component that keeps its [child] positioned according to the
+  /// Creates a component that keeps its [_child] positioned according to the
   /// [alignment] within this component's bounding box.
   ///
   /// More precisely, the child will be placed at [alignment] relative position
   /// within the current component's bounding box. The child's anchor will also
   /// be set to the [alignment], unless [keepChildAnchor] parameter is true.
   AlignComponent({
-    required this.child,
-    required Anchor alignment,
+    PositionComponent? child,
+    Anchor alignment = Anchor.center,
     this.widthFactor,
     this.heightFactor,
     this.keepChildAnchor = false,
-  }) {
+  }) : _child = child {
     this.alignment = alignment;
-    add(child);
+    this.child = child;
   }
 
-  late Anchor _alignment;
-
-  /// The component that will be positioned by this component. The [child] will
+  /// The component that will be positioned by this component. The [_child] will
   /// be automatically mounted to the current component.
-  final PositionComponent child;
```

Comment:
I would add a getter for `child` too and then refer to that getter in the dartdocs.